src/scss/pages/_games.scss

```diff
@@ -0,0 +1,145 @@
+.Games {
+    &-gallery {
+        align-items: center;
+        display: flex;
+        height: 723px;
+        justify-content: center;
+        width: 100%;
+
+        &-container {
+            height: 530px;
+            width: 1140px;
+        }
+
+        &-box {
+            display: flex;
+            height: 385px;
+        }
+
+        &-title {
+            color: $button-color;
+            display: flex;
+            font-size: 25px;
+            height: 144px;
+            justify-content: center;
+            width: 100%;
+        }
+
+        &-card {
+            display: flex;
+            height: 386px;
+            justify-content: center;
+            position: relative;
+            width: 285px;
+        }
+
+        &-image {
+            border-radius: 125px;
+            height: 340px;
+            width: 253px;
+        }
+
+        &-text {
+            align-items: center;
+            background: $opacity;
+            display: flex;
+            height: 61px;
+            justify-content: center;
+            opacity: 4;
+            position: absolute;
```

Comment:
Why are you using position absolute here?

Reply:
This is because this text is on top of an image and I couldn't even put it in the image, so the div of the image has relative position and the text has absolute position and that's how I manage to keep it on top of the image